### PR TITLE
Auto OCP-35068

### PR DESCRIPTION
**Pull request by xiuwang**
Passed locally.
@openshift/devexp-qe Please help review, thanks

### CODE REVIEW

File: lib/rules/web/images/jenkins_2/jenkins_2.xyaml [hunk 1]
```diff
@@ -593,7 +593,7 @@ jenkins_oauth_login:
   action: jenkins_verify_logged_in
 
 jenkins_multi_oauth_login:
-  url: /securityRealm/commenceLogin?from=%2F
+  url: /
```
Comment (liangxia) [inline lib/rules/web/images/jenkins_2/jenkins_2.xyaml:596]: @yapei  Could you help check ?  I'm not quite sure it will affect your team or not.

Comment (liangxia) [reply lib/rules/web/images/jenkins_2/jenkins_2.xyaml:596]: I'll merge later today if still no response.

**Comment by xiuwang** [comment]
@liangxia  Thanks Liang, this change will not affect UI, the rules/web/images/jenkins_2/jenkins_2.xyaml is only for jenkins feature.

**Pull request merged**